identity-wallet/Cargo.toml

```diff
@@ -47,10 +41,10 @@ typetag = "0.2"
 uuid = { version = "1.4", features = ["v4", "fast-rng", "serde"] }
 
 [dev-dependencies]
-indoc = "2.0"
-rand = "0.8"
-serial_test = "2.0"
+indoc.workspace = true
+rand.workspace = true
+wiremock.workspace = true
+serial_test.workspace = true
+tempfile.workspace = true
+
```

Comment:
`url` is not under **[dev-dependencies]** anymore?

Reply:
Was it removed completely as a dependency of `identity-wallet`?

Reply:
I looked at it and no more references where there